tasks/users_props.yml

```diff
@@ -14,8 +14,6 @@
     port: "{{ item.port | default(omit) }}"
     state: "{{ item.state | default('present') }}"
   with_items: "{{ postgresql_users }}"
-  environment:
-    PGOPTIONS: "{{ '-c password_encryption=scram-sha-256' if postgresql_auth_method == 'scram-sha-256' else omit }}"
   no_log: "{{ postgres_users_no_log }}"
   become: true
   become_user: "{{ postgresql_user }}"
```

Comment:
```
  environment:
    PGOPTIONS: "{{ (postgresql_auth_method == 'scram-sha-256') | ternary('-c password_encryption=scram-sha-256', '') }}"
```

This is missing. Rebasing this PR with the master branch (and consolidating to a single commit) I think would solve the issues with

>  Ensure PostgreSQL users are present